packages/common/src/projects/_internal/ProjectUiSchemaEdit.ts

```diff
@@ -68,6 +68,16 @@ export const uiSchema: IUiSchema = {
             },
           },
         },
+        {
+          labelKey: "{{i18nScope}}.fields.featuredImageAltText.label",
+          scope: "/properties/view/properties/featuredImageAltText",
+          type: "Control",
+          options: {
+            helperText: {
+              labelKey: "{{i18nScope}}.fields.featuredImageAltText.helperText",
```

Comment:
and youl'll want to make this `"{{i18nScope}}.fields.featuredImage.altText.helperText"`